website/templates/project/statistics.mako

```diff
@@ -2,34 +2,76 @@
 <%def name="title()">${node['title']} Analytics</%def>
 
 <div class="page-header  visible-xs">
-  <h2 class="text-300">Analytics</h2>
+    <h2 class="text-300">Analytics</h2>
 </div>
 
-<%
-    if user['is_contributor']:
-        token = user.get('piwik_token', 'anonymous')
-    else:
-        token = 'anonymous'
-
-    if node.get('piwik_site_id'):
-        piwik_url = '{host}index.php?module=Widgetize&action=iframe&moduleToWidgetize=Dashboard&actionToWidgetize=index&idSite={site_id}&period=day&date=today&disableLink=1&token_auth={auth_token}'.format(
-            host=piwik_host,
-            auth_token=token,
-            site_id=node['piwik_site_id'],
-        )
-%>
-
-% if not piwik_host or not node['piwik_site_id']:
-    <div class="row m-lg">
-        <div class="col-xs-12 text-center">
-            <img src="/static/img/no_analytics.png">
+<div class="row m-lg">
+    <div class="col-sm-4">
+        <div class="panel panel-default">
+            <div class="panel-heading clearfix">
+                <h3 class="panel-title">Visits</h3>
+            </div>
+            <div id="visits" class="panel-body">
+                <div class="text-center">
+                    <div class="logo-spin logo-lg"></div>
+                </div>
+            </div>
         </div>
     </div>
-% else:
-    % if not node['is_public']:
-        <div class='alert alert-warning'>
-            <strong>Note:</strong> Usage statistics are collected only for public resources.
```

Comment:
I'm an idiot.  I thought this was saying statistics are only *shown* for public resources.  You were correct to remove it.